src/libraries/Microsoft.PowerFx.Core/Public/Config/PowerFxConfig.cs

```diff
@@ -1,219 +1,228 @@
-﻿// Copyright (c) Microsoft Corporation.
```

Comment:
Formatter change? Why do we have so many changes in this file?

Reply:
Maybe a CR/LF issue, someone edited it on a Mac, I can't tell

Reply:
I confirm CR/LF issue.

Reply:
yeah, if I "ignore whitespace", then there's just a single diff